src/styles/header.scss

```diff
@@ -0,0 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: white;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, Arial, sans-serif;
+  text-transform: uppercase;
+}
+
+.menu__link {
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  height: 100%;
+  display: inline-block;
+  line-height: 60px;
+}
+
+.menu__link:hover {
+  color: var(--primary-color);
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.menu__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
+.menu__item {
+  height: 100%;
+}
+
+.menu__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active {
+  color: var(--primary-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: var(--primary-color);
+  position: absolute;
+  top: 56px;
+}
```

Comment:
The `.is-active::after` selector is used to add a line under the active menu item. However, according to the task, the line should be 2px high, not 4px. You should adjust the `height` property accordingly.